### PR TITLE
Add faceting to corporate name, genre and personal name subject types…

### PR DESCRIPTION
**Pull request by VivianChu**
…. DHH-829

### CODE REVIEW

**Comment by VivianChu** [comment]
@ucsdlib/developers - Please review, comment, etc...  Thanks

File: app/views/shared/fields/_corporate_name.html.erb [hunk 1]
```diff
@@ -15,14 +15,14 @@
...
 
-		fieldData.each do |datum|
-			corporateName = datum
-			concat "<li>#{corporateName}</li>".html_safe
-		end
-
-		concat htmlClose.html_safe
+	  <ul class="unstyled glyphicon-list glyphicon-list-tag">
+	    <% fieldData.each do |topic| %>
+	      <li><%= link_to topic, facet_uri(topic, 'subject_topic_sim') %></li>
```
Comment (mcritchlow) [inline app/views/shared/fields/_corporate_name.html.erb]: this goes for each of the additions of this block in the PR. Are these each still in the general `subject_topic_sim` facet or are they in their own?

Comment (VivianChu) [inline app/views/shared/fields/_corporate_name.html.erb]: @mcritchlow corporate_name, personal_name and genre are still in general subject_topic_sim.  Lithology, Cruise, Scientific Name and Common Name are in their own.

Comment (mcritchlow) [inline app/views/shared/fields/_corporate_name.html.erb]: 👍

**Comment by hweng** [comment]
:+1:

**Comment by lsitu** [comment]
:+1:

**Comment by lsitu** [comment]
:+1:

**Comment by VivianChu** [comment]
Thanks everyone for reviewing.  Could anyone please merge this PR?

**Comment by lsitu** [comment]
@VivianChu It's done.

**Comment by VivianChu** [comment]
Thanks @lsitu for merging

**Pull request merged**